src/5e-SRD-Monsters.json

```diff
@@ -235,32 +253,32 @@
             {
               "name": "Light",
               "level": 0,
-              "url": "http://www.dnd5eapi.co/api/spells/180"
```

Comment:
Huh. None of those are valid URLs. I must have missed those when I transferred spells over to string indices.

Reply:
This can be done in a separate PR though.